homeassistant/components/alert/__init__.py

```diff
@@ -52,6 +32,7 @@
     DOMAIN,
     LOGGER,
 )
+from .entity import AlertEntity
```

Comment:
Renaming this is potentially breaking for custom integrations, even though it's not meant for subclassing. Should we add an alias variable so both names are available?

Reply:
I didn't think it was needed.
I can cancel the rename if we think it might be in use somewhere... but I couldn't find any references to it in the core.

Reply:
It's custom integrations that may be affected.

Reply:
Quick check: 

![CleanShot 2024-09-16 at 20 04 44@2x](https://github.com/user-attachments/assets/880763ff-0014-4b0d-8fdf-9e6a894a8aff)

Haven't found a single custom integration (published in HACS) that would be affected by this.

Reply:
Ok, good.

Reply:
Thanks for checking @frenck - but did you check for the new name `AlertEntity` or the old name `Alert`?